Competition2017/src/competition/subsystems/autonomous/selection/AutonomousCommandSelector.java

```diff
@@ -0,0 +1,39 @@
+package competition.subsystems.autonomous.selection;
+
+import org.apache.log4j.Logger;
+
+import com.google.inject.Inject;
+
+import edu.wpi.first.wpilibj.command.Command;
+import xbot.common.command.BaseSubsystem;
+import xbot.common.properties.StringProperty;
+import xbot.common.properties.XPropertyManager;
+
+public class AutonomousCommandSelector extends BaseSubsystem {
+private static Logger log = Logger.getLogger(AutonomousCommandSelector.class);
+    
+    public final StringProperty currentAutonomousCommandName;
+    
+    Command currentAutonomousCommand;
+
+    @Inject
+    public AutonomousCommandSelector(XPropertyManager propManager) {
+        currentAutonomousCommandName = propManager.createEphemeralProperty("currentAutonomousCommandName", "No command set");
```

Comment:
Instead of having the "No command set" string duplicated, leave it blank/null here and then immediately call the `setCurrentAutonomousCommand` method.